pages/map.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Simple Map</title>
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
+    <style>
```

Comment:
Im cool with this style thing here cause its just for the map page ya know? Doesnt need its own file.